docs/theory/apportion_abundance.md

```diff
@@ -1,52 +1,70 @@
+B_{\textrm{M}}^{k} =
+\sum_{\ell} B_{\textrm{M}, \ell}^{k, \textrm{unaged}} +
+\sum_{\ell, \alpha} B_{\textrm{M}, \ell, \alpha}^{k, \textrm{aged}}
 \label{eq:biomass_M} \tag{1}
 $$
 
 $$
-B_{\textrm{F}}^{k} = \sum_{\textrm{F}, \ell, \alpha} B_{\textrm{F}, \ell, \alpha}^{k, \textrm{aged}} + \sum_{\textrm{F}, \ell, \alpha} B_{\textrm{F}, \ell, \alpha}^{k, \textrm{unaged}}
+B_{\textrm{F}}^{k} =
+\sum_{\ell} B_{\textrm{F}, \ell}^{k, \textrm{unaged}} +
+\sum_{\ell, \alpha} B_{\textrm{F}, \ell, \alpha}^{k, \textrm{aged}}
 \label{eq:biomass_F} \tag{2}
 $$
 
-Similarly, biomass estimates for all fish ($B^{k}$), which is inclusive of both sexed and unsexed fish, are also summed (see: <b>[kriged biomass summation for more details](apportion_biomass.md#total-biomass-apportioned-with-length-and-age)</b>).
+The biomass estimates for all fish including both sexed and unsexed fish in the transect interval $k$ is then:
+
+$$
+B^k = B_\textrm{M}^k + B_\textrm{F}^k.
+$$
 
 
-These kriged biomass estimates are then converted to sexed ($\hat{N}_{s}^{k}$) and total ($\hat{N}^{k}$) abundance by using averaged length-weight regression output ($\overline{W}(\ell)$). Consequently, $\overline{W}(\ell)$ can be defined either by using the average length-weight relationship produced or parameterizing $\overline{W}(\ell)$ with the mean length ($\bar{\ell}$). It is important to note, however, that both $\hat{N}_{s}^{k}$ and $\hat{N}^{k}$ are calculated using $\overline{W}(\ell)$ fit from <b>all</b> individuals (i.e. male, female, and unsexed).
+The estimated abundance $\hat{N}^k$ can be back-calculated from the kriged biomass estimates using an averaged length-weight relationship $\overline{W}(\ell)$ via:
 
 $$
-\hat{N}^{k} = \frac{B^{k}}{\overline{W}(\ell)}
+\hat{N}^{k} = \frac{B^{k}}{\overline{W}(\ell)},
 \label{eq:abundance} \tag{3}
 $$
 
+where $\overline{W}(\ell)$ is the length-weight regression relationship derived from the catch data. 
+
+Similarly, $\hat{\textit{NASC}^{k}}$ can be back-calculated from the estimated abundance using the averaged differential backscattering cross-section of the $i^{\text{th}}$ stratum, $\bar{\sigma}_{bs}^i$, via:
+
+$$
+\hat{\textit{NASC}^k} = \hat{N}^k \times \bar{\sigma}_\textrm{bs}^i,
+$$
+
+when the transect interval $k$ falls in stratum $i$. See [](stratification) for more information.
+
+
 ```{note} 
-With $\hat{N}_{\textrm{All}}^{k}$ calculated, $\hat{\textit{NASC}^{k}}$ can then be back-calculated by using the averaged $i^{\text{th}}$ differential backscattering cross-section ($\bar{\sigma}_{\textrm{bs}}$):
-$
-\hat{\textit{NASC}^{k}} = \hat{N}^{k} \bar{\sigma}_{\textrm{bs}}
-$
+In Chu's Echopro implementation, both $\hat{N}_{s}^{k}$ and $\hat{N}^{k}$ are calculated using a single $\overline{W}(\ell)$ fit from **all** (male, female, and unsexed) fish samples, instead of sex-specific fits.
```

Comment:
```suggestion
Both $\hat{N}_{s}^{k}$ and $\hat{N}^{k}$ are calculated using a single $\overline{W}(\ell)$ fit from **all** (male, female, and unsexed) fish samples, instead of sex-specific fits in the original `EchoPro` MATLAB implementation.
```

Reply:
I think that including this as a separate admonition is fine; however, the wording seems to imply a distinction with Echpop. If this were ever implemented in Echopop, we'd likely replicate the MATLAB code for consistency, no? I have no strong opinion on this one way or another, but just want to make sure we avoid implying there is a difference between `Echopop` and `EchoPro` here when they may not be one in the future.

Reply:
Yeah I agree with not implying either way, but am confused here -- wouldn't the changes you added make it actually different from EchoPro? since it says "instead of sex-specific fits in the original `EchoPro` MATLAB implementation"?

Reply:
>Yeah I agree with not implying either way, but am confused here -- wouldn't the changes you added make it actually different from EchoPro? since it says "instead of sex-specific fits in the original EchoPro MATLAB implementation"?

I didn't want to change the meaning of your suggested change too much without discussing that first.

```suggestion
Both $\hat{N}_{s}^{k}$ and $\hat{N}^{k}$ are calculated using a single $\overline{W}(\ell)$ fit from **all** (male, female, and unsexed) fish samples.
```